File: seed/csharp-sdk/exhaustive/explicit-namespaces/Snippets/Example37.cs
using SeedApi;
using SeedApi.Endpoints.HttpMethods;

public partial class Examples
{
    public async Task Example37() {
        var client = new SeedApiClient(
            token: "<token>",
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.Endpoints.HttpMethods.TestDeleteAsync(
            new TestDeleteHttpMethodsRequest {
                Id = "id"
            }
        );
    }

}

File: seed/java-sdk/exhaustive/custom-license/src/main/java/com/snippets/Example37.java
package com.snippets;

import com.seed.api.SeedApiClient;
import com.seed.api.resources.endpoints.httpmethods.requests.TestDeleteHttpMethodsRequest;

public class Example37 {
    public static void main(String[] args) {
        SeedApiClient client = SeedApiClient.builder()
                .token("<token>")
                .url("https://api.fern.com")
                .build();

        client.endpoints()
                .httpMethods()
                .testDelete("id", TestDeleteHttpMethodsRequest.builder().build());
    }
}
